test/e2e/e2e_suite_test.go

```diff
@@ -196,6 +197,7 @@ func initScheme() *runtime.Scheme {
 	Expect(controlplanev1.AddToScheme(sc)).To(Succeed())
 	Expect(bootstrapv1.AddToScheme(sc)).To(Succeed())
 	Expect(dockerinfrav1.AddToScheme(sc)).To(Succeed())
+	Expect(awsinfrav1.AddToScheme(sc)).To(Succeed())
```

Comment:
we add the API, but do we use it anywhere? it brings in quite a few dependencies in our go.mod, let's be mindful of those